addon/components/basic-dropdown/content.js

```diff
@@ -74,7 +94,12 @@ export default WormholeComponent.extend({
   },
 
   handleRootMouseDown(dropdownContent, e) {
-    let comesFromInside = this.element.parentElement.contains(e.target) || dropdownContent.contains(e.target);
+    if(this.get('hasMoved')){
```

Comment:
`this.hasMoved` is not a property where we case about observations, so it is save to get/set as any regular property, without using Ember.get/set.